hack/upgrade-k8s.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/env/bin bash
```

is more compatible and conventional.